storybook/main.js

```diff
@@ -60,11 +60,6 @@ module.exports = {
 		name: '@storybook/react-webpack5',
 		options: {},
 	},
-	features: {
-		babelModeV7: true,
```

Comment:
Babel mode is the default since v7, see https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#babel-mode-v7-exclusively